{
  "comments": [
    {
      "key": {
        "uuid": "bab921fb_69adea87",
        "filename": "ci-scripts/dlrnapi_promoter/dlrnapi_promoter.py",
        "patchSetId": 3
      },
      "lineNbr": 29,
      "author": {
        "id": 517
      },
      "writtenOn": "2018-05-30T05:56:12Z",
      "side": 1,
      "message": "Can be replaced for filename, line, _, _",
      "revId": "66a9c171ebbee0ff57b83ab0c534049e62bd84d5",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "bab921fb_49aa2672",
        "filename": "ci-scripts/dlrnapi_promoter/dlrnapi_promoter.py",
        "patchSetId": 3
      },
      "lineNbr": 277,
      "author": {
        "id": 517
      },
      "writtenOn": "2018-05-30T05:56:12Z",
      "side": 1,
      "message": "This will break the for statement, and no more hashes will be try, we just have to do a logger.exception and continue.",
      "revId": "66a9c171ebbee0ff57b83ab0c534049e62bd84d5",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "bab921fb_29b6d2dc",
        "filename": "ci-scripts/dlrnapi_promoter/dlrnapi_promoter.py",
        "patchSetId": 3
      },
      "lineNbr": 352,
      "author": {
        "id": 517
      },
      "writtenOn": "2018-05-30T05:56:12Z",
      "side": 1,
      "message": "Not catching exception where there are not needed and doing a proper loggin.exception will get as more info without a global exception handler.",
      "revId": "66a9c171ebbee0ff57b83ab0c534049e62bd84d5",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": false
    }
  ]
}